lib/tasks/config_files.rake

```diff
@@ -127,4 +127,14 @@ namespace :config_files do
   end
 
 
+  desc 'Produce a list of email addresses for which the MTA should reject messages at RCPT time'
+  task :generate_mta_rejection_list => :environment do
+    example = 'rake config_files:generate_mta_rejection_list FILE=/tmp/rejection_list.txt'
+    check_for_env_vars(['FILE'], example)
+    f = File.open(ENV['FILE'], 'w')
```

Comment:
I don't think this needs to handle files – just print to stdout and then use shell redirection if you want it in a file?

```
bundle exec rake config_files:generate_mta_rejection_list > /tmp/rejection_list.txt
```

Makes it easier to preview your list.